subsys/nrf_security/src/ssf_secdom/bounce_buffers.h

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2025 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef __BOUNCE_BUFFERS_H__
+#define __BOUNCE_BUFFERS_H__
+
+#include <nrf.h>
```

Comment:
nit: nrf.h should already be included by nrfx.h, so I don't think this is needed
```suggestion
```

Reply:
Evidently it is not included by nrfx.h.

In any case this header has nrf as a depenency, not nrfx, and it is good practice to include dependencies.

![image](https://github.com/user-attachments/assets/ac580b13-c84d-401e-9c6f-9e1325bf11e5)

Reply:
sorry, the error was from something else.

Only including nrfx.h does work.

fixed